code/__HELPERS/unsorted.dm

```diff
@@ -1394,3 +1394,33 @@ B --><-- A
  B
 
 */
+
+
+var/atom/movable/orbiting = null

```

Comment:
My thoughts being "lets group all orbiting code together" instead of "lets group all atom/movable code together"